<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosDelftFewsWidget</class>
 <widget class="QWidget" name="ReosDelftFewsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Delft-FEWS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ReosPlotWidget" name="mPlotWidget" native="true"/>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mLineEditFileName"/>
         </item>
         <item>
          <widget class="QToolButton" name="mFileButton">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="mToolBarWidget" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="mListView"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="mTextBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReosPlotWidget</class>
   <extends>QWidget</extends>
   <header>reosplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
